docs/azure/migration/appcat/visual-studio.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Use Azure Migrate application and code assessment for .NET with Visual Studio
+description: Learn how to assess .NET applications to evaluate their readiness to migrate to Azure with Visual Studio.
+ms.topic: conceptual
+ms.date: 11/09/2023
+author: codemillmatt
+ms.author: masoucou
+---
+
+# Analyze applications with Visual Studio
+
+Azure Migrate application and code assessment for .NET is a tool to assess .NET source code to identify replatforming and migration opportunities to Azure. It helps you modernize and replatform large-scale .NET applications through a broad range of transformations, use cases, and code patterns.
+
+This guide describes how to use the application and code assessment Visual Studio extension to both scan and interpret the results of the scan.
+
+If you have not installed the Visual Studio extension, please follow [these instructions first](./install.md).
+
+## Scan your application
+
+The application and code assessment lets you decide which projects in your solution to scan to identify migration opportunities to Azure. Follow these steps to scan your application.
+
+1. Open the solution containing the projects you want to migrate to Azure in Visual Studio 2022.
+1. Right click on any of the projects in the Solution Explorer window and select **Re-platform to Azure**.
+    ![Screenshot of the re-platform to Azure menu item in Visual Studio](./media/vs/replatform.png)
+1. The utility will start and give you the option to start a new analysis report or open an existing one. It will also display any recent analysis reports.
+    ![Screenshot showing the recent analysis reports](./media/vs/appcat-start-screen.png)
+1. Click on **New report** and it will display the projects in your solution in a treeview. It will give you an option to select which projects to analyze. You fill find projects pre-selected for you that are most likely to have compatibility issues. You can change the selection by checking or unchecking the boxes next to the projects.
```

Comment:
```suggestion
1. Click on **New report** and it will display the projects in your solution in a treeview. It will give you an option to select which projects to analyze. You will find web projects pre-selected for you and you can change the selection by checking or unchecking the boxes next to the projects.
```